### DISCUSSION TITLE
Huge chunks of the earth gets imported

### DISCUSSION THREAD

**Discussion by Vitatrollet** [eliemichel/MapsModelsImporter#228] [Q&A]
[](url
![mapsmodelimporter screenshot blender](https://user-images.githubusercontent.com/108431970/176544377-38eacedd-be60-4001-86b6-499e953bbf99.jpg)
)

Huge sphere-like parts of the earth seems to get imported into blender with my capture file of a small local area. Any idea whats going on?

**Comment by Vitatrollet** [comment]
Tried again with another capture from renderdoc and got it to work.

**Reply by palmavilluan** [reply]
I have the same issue. What did you so it worked finally?
